### PR TITLE
Feat action tags

### PR DESCRIPTION
**Pull request by mattsliv**
- Changed naming of action tags

### CODE REVIEW

File: ImageViewerAnnotate.php [hunk 1]
```diff
@@ -12,9 +12,8 @@
 
 class ImageViewerAnnotate extends \ExternalModules\AbstractExternalModule {
 
-    private $imageViewTag = "@IMAGEVIEW";
-    // private $imagePipeTag = "@IMAGEPIPE";
-    private $imagePipeTag = "@IMAGEANNOTATE";
+    private $imageViewTag = "@IMAGE-UPLOAD-VIEW";
```
Comment (mcmcgrath13) [inline ImageViewerAnnotate.php:15]: are hyphens kosher? (what have other action tags used for their separator if any?

for the view tag, how about `@IMAGE-ANNOTATE-UPLOAD` ? I think that better ties it to our module and the viewing is a side effect?

File: js/imageViewerAnnotate.js [hunk 2]
```diff
@@ -3,6 +3,10 @@ var IVEM = IVEM || {};
 // shows marker along with inserting marker data into text field
 function showMarkerArea(target, source, input) {
   const markerArea = new markerjs2.MarkerArea(source);
+  // Limit available markers
+  markerArea.availableMarkerTypes = [
```
[the remark this answers is not in the document]
Comment (mattsliv) [reply js/imageViewerAnnotate.js:7]: I branched this PR from PR #29 so the changes from the commits of the parent PR will appear

[the remark this answers is not in the document]
Comment (mcmcgrath13) [reply js/imageViewerAnnotate.js:7]: Ah, in that case can you change the PR base to `feat-styling` so this PR only shows the changes made here?

File: ImageViewerAnnotate.php (hunk 1 above)
Comment (mattsliv) [reply ImageViewerAnnotate.php:15]: looking at some of the default action tags, it looks like hyphens are kosher (see below)

image

`@IMAGE-ANNOTATE-UPLOAD` works for me!

**Review by mcmcgrath13 — APPROVED** [review]
:shipit:

**Pull request merged**